packages/react-composites/src/composites/CallComposite/components/SidePane/SidePane.tsx

```diff
@@ -0,0 +1,94 @@
+import { availableSpaceStyles, sidePaneStyles, sidePaneTokens } from '../../../common/styles/Pane.styles';
+import { useCloseSidePane, useSidePaneContext } from './SidePaneProvider';
+import { PeopleAndChatHeader } from '../../../common/TabHeader';
+import { hiddenStyles } from '../../../common/styles/Pane.styles';
+
+/** @private */
+export interface SidePaneProps {
+  mobileView?: boolean;
+
+  // legacy arguments to be removed in breaking change
+  disablePeopleButton?: boolean;
+  disableChatButton?: boolean;
+  onChatButtonClicked?: () => void;
+  onPeopleButtonClicked?: () => void;
+}
+
+/** @private */
+export const SidePane = (props: SidePaneProps): JSX.Element => {
+  const { headerRenderer, contentRenderer, activeSidePaneId, overrideSidePane } = useSidePaneContext();
+  const renderingHiddenOverrideContent = overrideSidePane?.hidden && overrideSidePane?.contentRenderer;
+  const renderingOnlyHiddenContent = renderingHiddenOverrideContent && !activeSidePaneId;
+
+  const paneStyles = renderingOnlyHiddenContent
+    ? hiddenStyles
+    : props.mobileView
+    ? availableSpaceStyles
+    : sidePaneStyles;
+
+  let Header =
+    (overrideSidePane?.headerRenderer && !overrideSidePane?.hidden
+      ? overrideSidePane?.headerRenderer
+      : headerRenderer) ?? EmptyElement;
+  /**
+   * Legacy code to support old behavior of showing chat and people tab headers on mobile side pane.
+   * To be removed in breaking change.
+   */
+  const overrideSidePaneId = overrideSidePane?.hidden ? undefined : overrideSidePane?.sidePaneId;
+  const { closePane } = useCloseSidePane();
+  if (props.mobileView && (overrideSidePaneId === 'chat' || activeSidePaneId === 'people')) {
+    // use legacy header
+    Header = () => (
+      <PeopleAndChatHeader
+        onClose={overrideSidePaneId === 'chat' ? props.onChatButtonClicked ?? noop : closePane}
+        activeTab={activeSidePaneId === 'people' ? 'people' : 'chat'}
+        // legacy arguments to be removed in breaking change:
+        disablePeopleButton={props.disablePeopleButton}
+        disableChatButton={props.disableChatButton}
+        onPeopleButtonClicked={activeSidePaneId === 'people' ? noop : props.onPeopleButtonClicked}
+        onChatButtonClicked={overrideSidePaneId === 'chat' ? noop : props.onChatButtonClicked}
+      />
+    );
+  }
+
+  const ContentRender = !overrideSidePane?.contentRenderer || overrideSidePane?.hidden ? contentRenderer : undefined;
+  const OverrideContentRender = overrideSidePane?.contentRenderer;
+
+  if (!ContentRender && !OverrideContentRender) {
+    return <EmptyElement />;
+  }
+
+  return (
+    <Stack verticalFill grow styles={paneStyles} data-ui-id="SidePane" tokens={props.mobileView ? {} : sidePaneTokens}>
+      <Header />
+      <Stack.Item verticalFill grow styles={paneBodyContainer}>
+        <Stack horizontal styles={scrollableContainer}>
+          {ContentRender && (
+            <Stack.Item verticalFill styles={scrollableContainerContents}>
```

Comment:
Should we maintain a single content variable and it should hold the value of latest content. Is there a purpose of maintaining content and OverrideContent?